websocket.go

```diff
@@ -220,18 +223,18 @@ func (c *Client) Send(dst *network.ServerIdentity, path string, buf []byte) ([]b
 		if err != nil {
 			return nil, NewClientError(err)
 		}
-		c.si = dst
+		c.connections[dest] = conn
 	}
 	defer func() {
 		if !c.keep {
-			c.Close()
+			c.Close(dest)
```

Comment:
Proposition:
- here: only close one connection, delete it from the map
- `Client.Close` - close all connections, return the `string`-concatenated error-messages, if any